packages/react/src/components/UIShell/SideNav.js

```diff
@@ -107,11 +109,19 @@ const SideNav = React.forwardRef(function SideNav(props, ref) {
         handleToggle(event, true);
       }
     };
+
     eventHandlers.onBlur = (event) => {
       if (!event.currentTarget.contains(event.relatedTarget)) {
         handleToggle(event, false);
       }
     };
+
+    eventHandlers.onKeyDown = (event) => {
+      if (match(event, keys.Escape)) {
+        handleToggle(event, false);
+        window.location.href = href;
```

Comment:
I think since `href` is not guaranteed to be defined since we removed the default, let's wrap this line in an `if(href){...}`